nodeup/pkg/model/tests/encryptionbuilder/aescbc/cluster.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: kops/v1alpha2
+kind: Cluster
+metadata:
+  creationTimestamp: "2016-12-10T22:42:27Z"
+  name: minimal.example.com
+spec:
+  kubernetesApiAccess:
+  - 0.0.0.0/0
+  channel: stable
+  cloudProvider: aws
+  configBase: memfs://clusters.example.com/minimal.example.com
+  etcdClusters:
+  - etcdMembers:
+    - instanceGroup: master-us-test-1a
+      name: master-us-test-1a
+    name: main
+  - etcdMembers:
+    - instanceGroup: master-us-test-1a
+      name: master-us-test-1a
+    name: events
+  kubeEncryptionConfig:
+    resources:
+    - resources:
+      - secrets
+      providers:
+      - aescbc:
+          keys:
+          - name: key1
+            secret: c2VjcmV0IGlzIHNlY3VyQ==
```

Comment:
What is this secret used for? Should we have this in a kops secret?

Reply:
That secret is used by the apiserver to encrypt/decrypt the data, so ideally I think it should be a secret/obscured somehow. What makes it tricky is users can specify an arbitrary number of keys for each provider (the apiserver will try them each in order when decrypting), so I'm not quite sure how that will work with kops secrets? I suppose users could create multiple secrets with `kops secret create aescbckey key1 c2Vj...`,   `kops secret create aescbckey key1 nf9D...`, or something similar. We could then swap in the actual secrets when nodeup runs?

Reply:
Or actually, just store the entire encryptionconfig in a secret? That might simplify things a bit.

Reply:
I would store the encryptionconfig in a secret.  We just had a PR for docker login perms.  Let me find it.

Reply:
https://github.com/kubernetes/kops/pull/3087